Cargo.toml

```diff
@@ -72,15 +72,17 @@ version = "0.16.0"
 [profile.release]
 # debug = 1 means line charts only, which is minimum needed for good stack traces
 debug = 1
+# Write debug info into a separate file.
 split-debuginfo = 'packed'
-strip = 'none'
+# Without stripping, sui binary size would be > 1GB.
+strip = 'debuginfo'
```

Comment:
Note that we still have symbols, so stack traces are still symbolized but it may miss info about inlined functions. Sui is using `strip = 'none'` until ~2 weeks ago. If debug info has been useful to you, you can still load the separate debug info file for gdb or jeprof.

Reply:
I have read that rather than have a separate file with only debug symbols, it is better to just have the full executable with more debug info as it is more reliable for various debug tooling.    IE, something like this:
1. build with full debug symbols
2. cp binary as `sui-node-with-debug`
3. run `strip` on original binary so it's all stripped down

Either way I think the inlined info is pretty important for good stack traces.

Reply:
Sure. I guess we can have a separate profile for non-stripped binaries and use that profile for devnet and testing.